### DISCUSSION TITLE
Welcome to ForceAutoHDR Discussions!

### DISCUSSION THREAD

**Discussion by 7gxycn08** [7gxycn08/ForceAutoHDR#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wff9527** [comment]
谢谢您的软件，帮我解决了难题，请问该怎么打赏您呢

**Reply by 7gxycn08** [reply]
在 GitHub 上给星

**Comment by aosfan** [comment]
Hello. Your program adds a registry key named `D3DBehaviors` that appears to be completely undocumented. I couldn't find any evidence in official sources that this key even exists, let alone that it has any effect. Could you please clarify where you obtained this key and what justifies its use?

**Reply by 7gxycn08** [reply]
D3DBehaviors is not a standard “single-purpose” Windows registry key. It’s a string value used by Direct3D / DirectX application override entries to apply specific compatibility, debugging, or feature flags to an individual program.

**Reply by aosfan** [reply]
Thank you for the explanation. If you don't mind me asking, where did you learn about the `D3DBehaviors` key? I understand that undocumented values like this sometimes come up in practice, but I'm genuinely curious what your decision to use it is based on — perhaps reverse engineering experience, internal testing, or some other information you could share? I'd appreciate any details you can provide.

**Reply by 7gxycn08** [reply]
It's a pretty common method usually used by game modding communities.
With internal testing it is confirmed to work properly.

**Comment by wff9527** [comment]
老大.软件对异环好像不起作用，添加了还是没有HDR

**Reply by wff9527** [reply]
异环今天更新之后可以了，应该是游戏的问题
